### ISSUE TITLE
Clean up demo box installation wiki

### ISSUE THREAD

**Issue by Bill-Keuntje** [rchain-community/liquid-democracy#72]

**Comment by dckc** [comment]
clean it up in what way? How can we tell when this is done?

**Comment by Bill-Keuntje** [comment]
> clean it up in what way? How can we tell when this is done?

This will be done when it is readable. Take a peek at it and the work required should be obvious.

**Comment by Bill-Keuntje** [comment]
@dckc @jimscarver @w2vy
Please review https://github.com/rchain-community/rgov/wiki/Demo-machine-setup-(Ubuntu) for readability. Comment and revise as needed.

**Comment by dckc** [comment]
There's nothing I need from it.

**Comment by Bill-Keuntje** [comment]
> There's nothing I need from it.

Thank you for your comments and revisions.

**Comment by w2vy** [comment]
It appears to have all the steps, but I fear they are focused on the manual way of adding things.
There are more automated ways of creating a site.
The ONE example I can see right off the bat is:
`cd sites-enabled
ln -s ../sites-available/demoapi.rhobot.net
ln -s ../sites-available/demoadmin.rhobot.net
`
On modern Ubuntu systems I think you can do
a2ensite demoapi.rhobot.net

I think this guide might have better longevity if it were refer to official installation (getting started) pages for nginx, letsencrypt, etc

I have in the past followed correct install steps for outdated versions and shot myself in the foot, fighting issues that were solved years realier.

**Comment by Bill-Keuntje** [comment]
> It appears to have all the steps, but I fear they are focused on the manual way of adding things.
> There are more automated ways of creating a site.
> The ONE example I can see right off the bat is:
> `cd sites-enabled ln -s ../sites-available/demoapi.rhobot.net ln -s ../sites-available/demoadmin.rhobot.net `
> On modern Ubuntu systems I think you can do
> a2ensite demoapi.rhobot.net
> 
> I think this guide might have better longevity if it were refer to official installation (getting started) pages for nginx, letsencrypt, etc
> 
> I have in the past followed correct install steps for outdated versions and shot myself in the foot, fighting issues that were solved years realier.

Forgive my ignorance here, but isn't a2ensite an Apache (not nginx) command?

**Comment by w2vy** [comment]
Yes that is true, but the point is: should we provide internal details of how to install a third party product or refer to the official docs?

Also is nginx a requirement or simple a webserver?
